.github/workflows/ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           - "--mapping_tool bowtie2"
           - "--skip_preprocessing"
           - "--run_mtnucratio --mitochondrion_header 'NC_007596.2'"
+          - "--run_contamination_angsd"
```

Comment:
```suggestion
          - "--run_mtnucratio --mitochondrion_header 'NC_007596.2' --run_contamination_angsd"
```

Reply:
We're trying to minimise the amount of tests, so adding all non-clashing parameters to one test, instead of running all possible combinations.